virttest/qemu_storage.py

```diff
@@ -986,9 +986,12 @@ def convert(self, params, root_dir, cache_mode=None,
         convert_cmd = self.image_cmd + " " + \
             self._cmd_formatter.format(self.convert_cmd, **cmd_dict)
 
-        LOG.info("Convert image %s from %s to %s", self.image_filename,
-                 self.image_format, convert_image.image_format)
-        process.run(convert_cmd)
+        timeout = params.get_numeric("conversion_timeout", 12000)
+
+        LOG.info("Convert image %s from %s to %s with timeout %s",
+                 self.image_filename, self.image_format,
+                 convert_image.image_format, timeout)
+        process.run(convert_cmd, timeout=timeout)
```

Comment:
Do we need do assert about process.run() exit status ?

Reply:
Thanks for taking a look @chunfuwen! In this case I would say "no" because the `process.run` would fail on nonzero status to begin with. This means that we will be able to detect the conversion did not go well the way we do with all other commands we run via the `process` module.